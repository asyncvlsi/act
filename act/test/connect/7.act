bool b;

b(4);
