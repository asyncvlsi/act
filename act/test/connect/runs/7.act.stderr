ERROR: File `7.act', line 3, col 5
	Connection specifier used for instance `b' whose root type is `bool'
	(not a user-defined type)
